server/elm/elm-watch.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# nice colors
+COLOR_OFF="\e[0m";
+DIM="\e[2m";
+
+# random filename for the lock; see below
+LOCKNAME=$(cat /dev/urandom | tr -cd 'a-f0-9' | head -c 16);
+
+function run {
+  (
+  flock 200; # don't let multiple `elm make` scripts run at once.
+
+  # reset the terminal scrollback history
+  # --> all the errors you see are the current ones, not stale
+  clear;
+  tput reset;
+
+  echo -en "${DIM}";
+  date -R;
+  echo -en "${COLOR_OFF}";
+
+  elm make src/Main.elm --output ../hedley/modules/custom/hedley_aggregated_ncda/js/elm-main.js
+  # on Linux optionally prepend for better performance: sysconfcpus -n 1
```

Comment:
We are able to conditionally invoke `sysconfcpus` if it's present.
Do you plan to use it inside DDEV or outside?

Reply:
```bash
if command -v sysconfcpus >/dev/null; then
    sysconfcpus -n 1 elm make src/Main.elm --output ../hedley/modules/custom/hedley_aggregated_ncda/js/elm-main.js
else
    elm make src/Main.elm --output ../hedley/modules/custom/hedley_aggregated_ncda/js/elm-main.js
fi

```

Reply:
Perhaps we might want to install that tool in `.ddev/web-build/Dockerfile`

Reply:
@AronNovak again, we got this from WFP-INCE.
On my local station current way of compiling works just fine.
It's very fast as well, so, I don't see a need for CPU optimizations right now.